go.mod

```diff
@@ -1,11 +1,14 @@
 module github.com/git-town/git-town/v16
 
-go 1.22
+go 1.22.0
+
+toolchain go1.22.3
```

Comment:
[govulncheck](https://pkg.go.dev/golang.org/x/vuln/cmd/govulncheck) reports a number of vulnerabilities in Go 1.22.3. Can you either update this to 1.22.5 or higher, or alternatively not add this line in this PR?

Reply:
done